gdal/swig/python/osgeo/__init__.py

```diff
@@ -139,3 +139,10 @@ def ver_str(ver):
     from warnings import warn, simplefilter
     simplefilter('always', DeprecationWarning)
     warn(msg, DeprecationWarning)
+
+
+try:
+    import numpy  # noqa
```

Comment:
I'm a bit ambivalent about automatically importing numpy. This could potentially have performance implications in situations where people don't need numpy.

Reply:
good point, I can think of these solutions:
1. https://stackoverflow.com/questions/14050281/how-to-check-if-a-python-module-exists-without-importing-it
```
import importlib
numpy_available = importlib.util.find_spec("numpy") is not None
```
2. put `numpy_available`  by itself in `osgeo/numpy_available.py`
3. as before, duplicate the try/except clause instead of importing `numpy_available`

Any preference?

Reply:
> Any preference?

probably 3

Reply:
done